remote-trait-object-macro/src/service.rs

```diff
@@ -43,32 +43,42 @@ impl<T: Parse> Parse for SingleArg<T> {
 #[derive(Default)]
```

Comment:
The commit message " Revise macro arguments " looks too general.
How about using a commit message that explains what you did like "Make macro be able to generate server side only".

Reply:
I updated